examples/MANUAL.md

```diff
@@ -101,6 +101,16 @@ is provided so the application can be easily adapted to any environment.
         This mode can also be combined with a "test database" in InfluxDB to
         give yourself a "test config file" you may run ad-hoc to test changes.
 
+        * Value: prometheus
+        In this mode the application opens an http interface and exports the
+        measurements at /metrics for collection by prometheus. Enabling this
+        mode disables InfluxDB usage entirely. This is BETA.
+
+    http_listen            default: 0.0.0.0:61317
```

Comment:
You can register on the [Prometheus port registry](https://github.com/prometheus/prometheus/wiki/Default-port-allocations) if you like. Or maybe re-use 9130, which is allocated to the current unifi exporter.

Reply:
9130 sounds like a great idea, I'll do that!